docs/docs/workflows/troubleshooting.md

```diff
@@ -149,3 +149,50 @@ Debugging via a shell on a node is [directly supported by Kubernetes](https://ku
     ```bash
     kubectl delete pod node-debugger-constell-worker-xksa0-000000-bjthj
     ```
+
+### Emergency SSH access
+
+Emergency SSH access to nodes can be useful to diagnose issues or download important data even if the Kubernetes API isn't reachable anymore.
+
+1. Enter the `constellation-terraform` directory in your Constellation workspace and enable emergency SSH access to the cluster:
+
+   ```bash
+    cd constellation-terraform
+    echo "emergency_ssh = true" >> ./terraform.tfvars
+    terraform apply
+   ```
+
+2. Sign an existing SSH key with your master secret:
+
+   ```bash
+   cd ../ # go back to your constellation workspace
+   constellation ssh --key your_public_key.pub
+   ```
+
+   A certificate is written to `constellation_cert.pub`.
+
+   The certificate is valid for 24 hours and enables you to access your constellation nodes using
+   [certificate based authentication](https://en.wikibooks.org/wiki/OpenSSH/Cookbook/Certificate-based_Authentication).
+
+3. Finally, you can connect to any constellation node using your certificate and your private key.
+
+   `ssh -o CertificateFile=constellation_cert.pub -i <your private key> root@<ip of constellation node>`
+
+   Normally, you won't have access to all constellation nodes since they reside in a private network.
+   To access those nodes anyways, you can use your Constellation load balancer as a proxy jump host.
+   For this, use something along the following ssh client configuration:
```

Comment:
```suggestion
   Normally, you don't have access to the Constellation nodes since they reside in a private network.
   To access those nodes, you can use your Constellation load balancer as a proxy jump host.
   For this, use something along the following SSH client configuration:
```